app/src/main/java/com/snaggly/ksw_toolkit/core/service/mcu/McuSenderInterceptor.java

```diff
@@ -21,9 +21,7 @@ public void startReading(McuCommunicator.McuAction notifier) {
         if (getReading()) return;
         Thread readerThread = new Thread(() -> {
             try {
-                //Commented out to check the light status by KSW before starting KSW-ToolKit
-                //Return if there is a problem with startup
-                //Runtime.getRuntime().exec("logcat -c\n");
```

Comment:
This is a comment-out I made in the past, but when I checked the behavior of Night Brightness again, this had nothing to do with it, so I reverted it back to its original state.